pkg/reconciler/instances/istio/actions/performer_test.go

```diff
@@ -292,6 +292,29 @@ func Test_DefaultIstioPerformer_PatchMutatingWebhook(t *testing.T) {
 		require.Contains(t, gotNew.Webhooks[0].NamespaceSelector.MatchExpressions, want)
 		require.NotContains(t, gotOld.Webhooks[0].NamespaceSelector.MatchExpressions, want)
 	})
+
+	t.Run("patching mutating webhook should be idempotent", func(t *testing.T) {
```

Comment:
In general I'm OK with this description but I would rather call define more verbosely such as:
`should patch mutating webhook only once when patch was performed twice` or something similar.